autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -27,21 +27,33 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
   const makeList = (): JSX.Element[] => {
```

Comment:
One last thing I would say is to add some comments to this function (especially about how the `ul` wrappers make it so the instructor groups slide in/out), this component is getting a little complicated so it would be nice to have some documentation in case we make changes in the future

Reply:
Done